### DISCUSSION TITLE
Unsupported Architecture: "ia32"

### DISCUSSION THREAD

**Discussion by on1977** [typegoose/mongodb-memory-server#845] [Q&A] [answered]
When  creating MongoMemoryServer, the error appears ' Unsupported Architecture: ia32'.

My laptop is: 12th Gen Intel(R) Core(TM) i3-1215U   1.20 GHz (64 bits) with widnwos 11 64 bits.

Im usingy  mongodb-memory-server@9.1.4.

**Comment by hasezoey** [comment]
is your nodejs maybe the ia32 (32-bit) version instead of 64 bit?

**Reply by on1977** [reply]
possibly yes, node -p "process.arch" gives ia32. So I have to install 64bit nodejs? :)

**Reply by on1977** [reply, marked as answer]
Thanks, after installing 64-bit nodejs, error disappeared.
